src/index.js

```diff
@@ -120,11 +120,12 @@ const argon2u = async (entropy, ship) => {
  * SHA-256 hash function.
  *
  * @param  {Array, ArrayBuffer, Buffer, String} args any number of arguments
- * @return {Promise<ArrayBuffer>}  the hash, as bytes
+ * @return {<Buffer>}  the hash, as bytes
```

Comment:
Surely there's no weird js stuff that can go wrong with this. (^:

Reply:
Haha.  I think it was fine, but I took another pass over everything just to be sure.. 😄 

The functions that use `sha256` (and the function that use them, etc.) can be de-async'd, so I've done that.  Also there was one superfluous use of `Buffer.from` so best nix that as well.

Reply:
Thanks! This still looks good to me. (: